index.html

```diff
@@ -0,0 +1,99 @@
+<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+<!DOCTYPE html>
+<head>
+  <h1 id="title">Amazon Survey Form</form></h1>
```

Comment:
Use the appropriate tags for each element (e.g., links, titles, etc.) and use HTML5 semantic tags (e.g., header, nav) over divs to makes the HTML more comprehensible by better defining the different sections and layout of web pages.